cmd/mdatagen/templates/readme.md.tmpl

```diff
@@ -7,10 +7,13 @@
 | Supported pipeline types | {{ stringsJoin .Status.Pipelines "," }}   |
```

Comment:
", "
```suggestion
| Supported pipeline types | {{ stringsJoin .Status.Pipelines ", " }}   |
```

Reply:
I'll commit this in a separate pr, as it changes many more readme's

Reply:
PTAL at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/20244